hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -502,6 +502,7 @@ public long numNftsOwnedBy(AccountID target) {
 			var adminKey = JKey.mapJKey(contract.getAccountKey());
 			info.setAdminKey(adminKey);
 		} catch (Exception ignore) {
+			return Optional.of(info.build());
```

Comment:
nit : may be we can log a message instead of a return here.

Reply:
Not sure if it will emit lot of logs, if a log is added(as the exception is ignored previously)